CHANGELOG.md

```diff
@@ -2,9 +2,8 @@
 
 ## master / unreleased
 
-## 1.9.0 in progress
+## 1.9.0-rc.0 / 2021-04-28
 
-* [CHANGE] Fix for CVE-2021-31232: Local file disclosure vulnerability when `-experimental.alertmanager.enable-api` is used. The HTTP basic auth `password_file` can be used as an attack vector to send any file content via a webhook. The alertmanager templates can be used as an attack vector to send any file content because the alertmanager can load any text file specified in the templates list. #4129
```

Comment:
why remove this? I think it's useful to keep it here for users to see that it's fixed on master too. WDYT?

Reply:
Do we usually put changes that already made it into a previous release into the new release changelog?

Reply:
You’re right, usually not.